sdk/communication/azure-communication-administration/CHANGELOG.md

```diff
@@ -40,7 +43,7 @@ accessKey(String accessKey) within CommunicationIdentityClientBuilder.
 Added phone number administration. For more information, please see the [README][read_me] and [documentation][documentation].
 
 ## 1.0.0-beta.1 (2020-09-22)
-This is the initial release of Azure Communication Administration, which manages users and tokens for Azure Communication Services. For more information, please see the [README][read_me] and [documentation][documentation].
+This is the initial release of Azure Communication Administration. For more information, please see the [README][read_me] and [documentation][documentation].
```

Comment:
We should not change the old version's log. That version was doing user management.

Reply:
Good point, I got confused. Reverted these changes and the ones above to 1.0.0.beta-3